scalafmt-core/shared/src/main/scala/org/scalafmt/config/Newlines.scala

```diff
@@ -168,14 +168,8 @@ case class Newlines(
     topLevelStatementsMinBreaks: Int = 1,
     topLevelStatements: Seq[BeforeAfter] = Seq.empty,
     beforeTemplateBodyIfBreakInParentCtors: Boolean = false,
-    templateBodyIfMinStatements: Seq[BeforeAfter] = Seq.empty,
-    templateBodyMinStatements: Int = 2,
-    @annotation.DeprecatedName(
-      "alwaysBeforeTopLevelStatements",
-      "Use newlines.topLevelStatements instead",
-      "2.5.0"
-    )
-    private val alwaysBeforeTopLevelStatements: Boolean = false,
+    topLevelBodyIfMinStatements: Seq[BeforeAfter] = Seq.empty,
```

Comment:
Should we still support `templateBodyIfMinStatements` just in case?

Reply:
i assumed that since we added this option recently, people who may have used it with one of the `RC` previews will simply rename it. is there any reason to keep this intermediate name?

Reply:
Ach ok, I forgot about that!